core/dom/range.js

```diff
@@ -2818,6 +2818,114 @@ CKEDITOR.dom.range = function( root ) {
 			reference.remove();
 		},
 
+		/**
+		 * Returns an array of {@link CKEDITOR.dom.rect} elements that are represented as rectangles which are covered by ranges.
+		 * For each DOM element within the selection range there is only one rectangle.
+		 * Rectangles represent the area of the screen occupied by the elements contained within the range.
+		 *
+		 * In the following example:
+		 *
+		 * ```html
+		 *  <p><span>first { span</span><span>second span</span></p>
+		 *  <p><span>very long } span</span></p>
```

Comment:
For some reason JSDuck wont create codeblock here if I don't put two spaces or two tabulations here. Couldn't fix that even with help from @msamsel.